openshift/machine-config-operator
# Decompress and decode Ignition configs from ConfigMap

**- What I did**

This will provide the MCD portion of the decompress / decode functionality required by [MCO-260](https://issues.redhat.com/browse/MCO-260). There is some additional work that will be needed on the Hypershift side, however this PR can merge before that does.

**- How to verify it**

I've added unit tests that can be run. Additional e2e tests which involve the Hypershift path should also be run against this.

**- Description for the changelog**

Support gzipped Ignition configs
